src/opts.c

```diff
@@ -758,6 +786,12 @@ natsOptions_SkipServerVerification(natsOptions *opts, bool skip)
     return nats_setError(NATS_ILLEGAL_STATE, "%s", NO_SSL_ERR);
 }
 
+natsStatus
```

Comment:
Would need to be remove, because this is what `NATS_HAS_TLS` is not defined, so `SSL_verify_cb` will be unknown.